guides/v2.3/release-notes/ReleaseNotes2.3.0Commerce.md

```diff
@@ -39,7 +39,7 @@ Magento Commerce 2.3.0 includes a wealth of new features as well as hundreds of
 
 * **Bulk Web APIs**  allow all existing REST APIs to accept payloads with multiple entities. These community-contributed bulk APIs support more efficient and scalable implementations that eliminate round-trip network overhead. Like asynchronous APIs, bulk web APIs can be used in conjunction with queues that have also been migrated to Magento Open Source. [See Bulk endpoints](https://devdocs.magento.com/guides/v2.3/rest/bulk-endpoints.html) for more information. 
 
-* **Updates to Magento's tech stack (including upgraded PHP support to maintain PCI compliance)** include upgrades to Redis, MySQL, Elasticsearch, compatibility with PHP 7.2.x. 
+* **Updates to Magento's tech stack (including upgraded PHP support to maintain PCI compliance)** include upgrades to Redis, MySQL, Elasticsearch and compatibility with PHP 7.2.x. 
```

Comment:
We use the Oxford, or serial, comma in a series. So in this instance, there would be a comma after "Elasticsearch" too. I will make this change but please note it for the future.